### ISSUE TITLE
Larger UI

### ISSUE THREAD

**Issue by moritzwiechers** [alanwoolley/CorsixTH-Android#133]
Hey Alan,
thanks for your great work. I love to play games i played as a child. TH is one of them. Playing it on my mobile is awesome. It works like a charm.

The only thing I'm struggling with is the touch handling. The ingame elements like rows in the build menu are very small, even using the smallest resolution.
Is it possible to enlarge them?
Greetings
Moritz

**Comment by alanwoolley** [comment]
I agree that this is a substantial problem with playing on a phone, even the larger ones on the market. It's much more playable on a foldable or tablet.

This isn't a simple one to solve, and I think a "proper" solution will be dependent on UI scaling implemented in the engine. Simple integer-based scaling is being tracked in https://github.com/CorsixTH/CorsixTH/issues/1210, but is unlikely to land any time soon.

As far as what can be done specifically for the Android port, that leaves us with a few options:
- Manually adjusting the particular windows that are most affected, however this will cause those areas to diverge in the fork and will then require additional effort to maintain. It also won't be possible to adjust all elements to be larger, such as those depending heavily on the original assets.
- Implement a way to "zoom" in on the UI - i.e. scale the whole game window. Pinch-to-zoom is already implemented for the map, and so a different gesture would have to be used.
- Disable fixed aspect ratio. This will result in UI being "stretched" along the longer axis, which will obviously increase the overall size of the elements, but won't look great especially the further away your device is from the "ideal" 4:3 ratio. The option to disable this is in the settings, but it's been broken since the application was largely rewritten in the v1.0 snapshots.
